src/components/navbar/navbar.js

```diff
@@ -120,7 +143,7 @@ export class Navbar extends Component {
         >
 
           <Image centered size='small' src='/assets/images/logo.png'/>
-          {this.createSideMenu()}
+          {this.createSideMenu(sideMenuContent)}
```

Comment:
isn't sideMenuContent imported in the file already?

Reply:
This part of the code is definitely not good and I will definitely change it later (all read menu from JSON stuff).
It was quite confusing to follow the flow, and that is why I added the comments.

I personally think that passing sideMenuContent (the JSON) explicitly was just a little bit more clear. So yes it is already imported and does not need to be passed explicitly.